.github/workflows/run_coverage.yml

```diff
@@ -0,0 +1,57 @@
+name: Run Coverage
+on: [pull_request]
+jobs:
+  test:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install poetry
+        run: |
+          pip install poetry
+      - name: Syncify Client
```

Comment:
Shouldn't run this - should check with the current state sync the syncified code won't be committed here